makefiles/gitea.mk

```diff
@@ -0,0 +1,57 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += gitea
+GITEA_VERSION := 1.14.3
+DEB_GITEA_V   ?= $(GITEA_VERSION)
+
+gitea-setup: setup
+	-[ ! -f "$(BUILD_SOURCE)/gitea-$(GITEA_VERSION).tar.gz" ] && \
+		wget -q -nc -O$(BUILD_SOURCE)/gitea-$(GITEA_VERSION).tar.gz \
+			https://github.com/go-gitea/gitea/releases/download/v$(GITEA_VERSION)/gitea-src-$(GITEA_VERSION).tar.gz
+	-[ ! -f "$(BUILD_WORK)/gitea" ] && \
+		mkdir -p $(BUILD_WORK)/gitea && \
+			tar xf $(BUILD_SOURCE)/gitea-$(GITEA_VERSION).tar.gz -C $(BUILD_WORK)/gitea
```

Comment:
The release src tarball contains the frontend files already compiled, this removes the build. dependency on nodejs that would exist otherwise. This tarball also doesn't have a prefix, hence I used tar directly instead of using `EXTRACT_TAR`.